spec/rspec/matchers/built_in/output_spec.rb

```diff
@@ -159,6 +159,14 @@ def print_to_stream(msg)
       }
     end
 
```

Comment:
Side observation - let(:macher) on line 6 of this spec seems unnecessary as it’s overridden by a later define_method.

May i kindly ask you to try and remove it?

Reply:
mmmm 10 specs did actually fail when I removed it, I don't really understand why. I'll leave that change out.